datasource/packer-image-iteration/data.go

```diff
@@ -122,7 +122,7 @@ type DatasourceOutput struct {
 	CreatedAt string `mapstructure:"created_at"`
 	// A list of builds that are stored in the iteration. These builds can be
 	// parsed using HCL to find individual image ids for specific providers.
-	Builds []ParBuild `mapstructure:"builds"`
+	Builds map[string]ParBuild `mapstructure:"builds"`
```

Comment:
Are these changes needed yet?

Reply:
ah, must have accidentally re-added them in a squash. I'll clean up now.